app/src/main/java/org/openobservatory/ooniprobe/common/ResubmitTask.java

```diff
@@ -35,18 +35,19 @@
      */
     public ResubmitTask(A activity) {
         super(activity, true, false);
+        // FIXME: we should use MKResources.
+        task = new MKReporterTask(
+                activity.getString(R.string.software_name),
+                BuildConfig.VERSION_NAME,
+                activity.getCacheDir() + "/" + Application.CA_BUNDLE
+        );
```

Comment:
This line should be changed to use `MKResources` as part of the more general endeavour described in #252.